tests/integration/targets/postgresql_privs/tasks/postgresql_privs_general.yml

```diff
@@ -653,6 +653,35 @@
     login_user: "{{ db_user3 }}"
     login_password: password
 
+# Issue https://github.com/ansible-collections/community.general/issues/994
```

Comment:
Not sure this comment is needed. Git blame will help you to find the commit id and the PR.

Reply:
yes, it's not needed but i'd leave this here because it helps me recognize things faster and later the postgres-related stuff might be moved to a separate repo, so `git blame` would show that this was created by a person who moved the content. would be helpful in this case, i think.